packages/block-editor/src/hooks/use-resize-canvas.js

```diff
@@ -68,3 +70,9 @@ export function useResizeCanvas() {
 
 	return contentInlineStyles( deviceType );
 }
+
+addFilter(
```

Comment:
Is this a new filter, did we have it before?

Reply:
Yes, it's a new filter.

Reply:
We've been avoiding adding new filters in Gutenberg as we figured that filters and hooks are not the best API for extensibility in JS. 

Could you clarify why this filter is needed and do you think we should have it added in its own PR instead? We could discuss alternatives... for your use cases.

Reply:
I don't think it's needed, it's just that I noticed that that's how hooks are usually organized/used (haven't worked with them so far in Gutenberg), so I figured that a request to move this to hooks folder would imply a filter.

Reply:
Ohhh, I guess we have a naming conflicts (react hooks vs WP hooks) 🤔

Reply:
But now I realize that my assumption has been wrong 😅. I pushed a revert for that in 09cfe0e673c9362d78ce409cb5f186a7abb8d1ef.